### PR TITLE
 Run metering on CREATE output only if it returned wasm bytecode

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

**Comment by axic** [comment]
@hugo-dc this should fix your issue

File: src/hera.cpp [hunk 1]
```diff
@@ -56,6 +56,10 @@ struct hera_instance : evm_instance {
 
 namespace {
 
+bool hasWasmSignature(vector<uint8_t> const& _input) {
```
Comment (chfast) [inline src/hera.cpp]: Not sure, but maybe "magic number" would be a better name instead of "signature". Or "prefix". Or `isWasmCode()`.

Comment (axic) [reply src/hera.cpp]: It is called signature in the spec IIRC.

Comment (axic) [reply src/hera.cpp]: According to the spec, we should extend that to 8 bytes and called is preamble. The version byte is little endian 32 bits, so the next 3 bytes would be zero.

**Pull request merged**